src/org/opendatakit/briefcase/util/ExportToCsv.java

```diff
@@ -134,6 +136,46 @@ public boolean doAction() {
 
     File[] instances = instancesDir.listFiles();
 
+		// Sorts the instances by the submission date. If no submission date, we
+		// assume it to be latest.
+		Arrays.sort(instances, new Comparator<File>() {

```

Comment:
Instances could be null at this point so it would be good to do a null check before sorting.